ambari/src/main/java/io/brooklyn/ambari/server/AmbariServer.java

```diff
@@ -58,7 +58,7 @@
     AttributeSensor<String> CLUSTER_STATE = Sensors.newStringSensor("ambari.server.clusterState");
 
     BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey USERNAME =
-            new BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey("ambari.server.http.user", "Ambari Server's http username", "admin");
+            new BasicAttributeSensorAndConfigKey.StringAttributeSensorAndConfigKey("ambari.server.http.user", "Ambari Server's http username. It is always admin", "admin");
 
```

Comment:
Seems daft to have a config key that has no effect when it is set - should we just delete this?